### PR TITLE
feat: add more plain methods

### PR DESCRIPTION
**Pull request by anho**
Slightly deviated from the current naming schema for organizations because its not paginated.

### CODE REVIEW

File: lib/plain/plain-client.ts [hunk 1]
```diff
@@ -23,6 +26,7 @@ export const createPlainClient = (params: ClientParams, defaults?: DefaultParams
     },
     user: {
       getManyForSpace: wrap(wrapParams, endpoints.user.getManyForSpace),
+      getCurrent: wrapHttp(http, endpoints.user.getCurrent),
```
Comment (suevalov) [inline lib/plain/plain-client.ts:29]: I think we can still use `wrap` and it's up to a method use or not to use params?

**Comment by suevalov** [comment]
@anho you need to use "feat: " prefix for the PR if you want to trigger publish once it's merged

File: lib/plain/plain-client.ts (hunk 1 above)
Comment (anho) [reply lib/plain/plain-client.ts:29]: There would be a mandatory `params` popping up in the typings.

Comment (suevalov) [reply lib/plain/plain-client.ts:29]: got it

**Pull request merged**